### PR TITLE
Move GINKGO_FOCUS=BPF-SAFE to Makefile fv-bpf target

### PR DESCRIPTION
**Pull request by neiljerram**
GINKGO_FOCUS=BPF-SAFE is part of the definition of 'make fv-bpf', not
just of how we run under Semaphore

## Todos
- [ ] Unit tests (full coverage)
- [ ] Integration tests (delete as appropriate) In plan/Not needed/Done
- [ ] Documentation
- [ ] Backport
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -424,7 +424,7 @@ fv fv/latency.log fv/data-races.log: $(REMOTE_DEPS) image-test bin/iptables-lock
 	fi
 
 fv-bpf:
-	$(MAKE) fv FELIX_FV_ENABLE_BPF=true
+	$(MAKE) fv FELIX_FV_ENABLE_BPF=true GINKGO_FOCUS=BPF-SAFE
```
Comment (fasaxc) [inline Makefile:427]: Can this be overridden from the command line?

Comment (neiljerram) [reply Makefile:427]: Good point, I'll fix that.

**Comment by caseydavenport** [comment]
@fasaxc @neiljerram do we still want this? Looks like there is a conflict now.

**Comment by neiljerram** [comment]
No, I think things have moved on. Thanks for checking!

**Pull request closed without merging**